README.md

```diff
@@ -99,11 +98,9 @@ This template supports sending logs flowing to a S3 bucket to Honeycomb.
 
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?templateURL=https://honeycomb-builds.s3.amazonaws.com/cloudformation-templates/latest/s3-logfile.yml&stackName=Honeycomb-S3-Logs" target="_blank"><img src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png" alt="Launch Stack" /></a>
 
-
 ### Required Inputs
```

Comment:
@jharley When would these required inputs be encountered? In an AWS form after clicking the button?

Reply:
They're displayed in the UI (along with a lengthy list of optional parameters) or would need to be provided via the CLI or API (CloudFormation supports three avenues)

Reply:
Thank you! So an applicable description to add above each bulleted list could be...?
> Enter a value for each required input in the UI, or if using the CLI or API, ensure to include each required input and its value.

Reply:
Yep, that works!

Reply:
Added in the most recent commit. Thanks!